app/services/to_form/note_mapper.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ToForm
+  # Maps notes.
+  class NoteMapper
+    ABSTRACT_TYPE = 'abstract'
+    CITATION_TYPE = 'preferred citation'
+
+    def self.call(...)
+      new(...).call
+    end
+
+    def self.abstract(cocina_object:)
+      new(cocina_object:, type: ABSTRACT_TYPE).call
+    end
+
+    def self.citation(cocina_object:)
+      new(cocina_object:, type: CITATION_TYPE).call
+    end
```

Comment:
I guess you deviated from the `.call` pattern here for DRY reasons? I expected `ToForm::AbstractMapper.call` for instance.

Reply:
There is a `.call` method here. These are just for convenience (and an alternative to adding a class hierarchy).